src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts work");

```

Comment:
The message 'Truck starts work' could be more informative by specifying what type of work the Truck is starting. For example, 'Truck starts its delivery route' or 'Truck begins transportation work' would give more context.